examples/gce/README.md

```diff
@@ -11,6 +11,8 @@ This examples runs kube-lego in a separate namespace
 kubectl create -f 05-kube-lego-namespace.yaml
 # Deployment
 kubectl create -f 50-kube-lego-deployment.yaml
+# ConfigMap
+kubectl create -f 20-kube-lego-configmap.yaml
```

Comment:
I think this has to go on the line before the deployment as the deployment references items in the config map